recipes/nbpresent/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set version = "3.0.1" %}
+{% set gh_org = "Anaconda-Platform" %}
```

Comment:
This was based on doing a bunch of them on another repo... I can take it out, though it _has_ changed a fair number of times: bollwyvl, continuumio, anaconda-server... ah, memories.